src/ripple/consensus/Validations.h

```diff
@@ -580,17 +584,17 @@ class Validations
             // Check that validation sequence is greater than any non-expired
             // validations sequence from that validator
             auto const now = byLedger_.clock().now();
-            SeqEnforcer<Seq>& enforcer = seqEnforcers_[key];
+            SeqEnforcer<Seq>& enforcer = seqEnforcers_[nodeID];
             if (!enforcer(now, val.seq(), parms_))
                 return ValStatus::badSeq;
 
             // This validation is a repeat if we already have
-            // one with the same id for this key
-            auto const ret = byLedger_[val.ledgerID()].emplace(key, val);
-            if (!ret.second && ret.first->second.key() == val.key())
-                return ValStatus::repeatID;
+            // one with the same id and signing key for this node
```

Comment:
Should the comment be removed or modified to say that we don't reject a validation with the same id as a previous one with a lower sequence?

Reply:
Oops, forgot to remove.